src/JB721TiersHookStore.sol

```diff
@@ -874,6 +895,28 @@ contract JB721TiersHookStore is IJB721TiersHookStore {
         }
     }
 
+    /// @notice Records the setting of a discount for a tier.
+    /// @param tierId The ID of the tier to record a discount for.
+    /// @param discountPercent The new discount percent being applied.
+    function recordSetDiscountPercentOf(uint256 tierId, uint256 discountPercent) external override {
+        // Make sure the discount percent is within the bound.
+        if (discountPercent > JB721Constants.MAX_DISCOUNT_PERCENT) revert DISCOUNT_PERCENT_EXCEEDS_BOUNDS();
+
+        // Get a reference to the stored tier.
+        JBStored721Tier storage storedTier = _storedTierOf[msg.sender][tierId];
+
+        // Parse the flags.
+        (,,,, bool cannotIncreaseDiscountPercent) = _unpackBools(storedTier.packedBools);
```

Comment:
looks kinda crazy but in each of these unpack instances

`(
        /* bool ignoredVar */,
        /* bool ignoredVar2 */,
        /* bool ignoredVar3 */,
        /* bool ignoredVar4 */,
        bool cannotIncreaseDiscountPercent) = _unpackBools(storedTier.packedBools);`
        
    
is more readable and safe to me

Reply:
yeaahhhhhhh.